src/packet_analysis/Manager.h

```diff
@@ -163,6 +171,10 @@ class Manager : public plugin::ComponentManager<Tag, Component>
 	uint64_t unknown_sampling_rate = 0;
 	double unknown_sampling_duration = 0;
 	uint64_t unknown_first_bytes_count = 0;
+
+	FuncPtr packet_not_processed_hook = nullptr;
+	uint64_t total_not_processed = 0;
+	iosource::PktDumper* unprocessed_dumper = nullptr;
```

Comment:
Mostly curious - is this deleted during shutdown? (I know, there is a ton of stuff that is not - but I wonder if, for new things, we should).

This might honestly not be worth addressing though.

Reply:
The IOSource manager keeps track of all of the open dumpers and destroys them when it shuts down.